hack/tools/go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/operator-framework/rukpak/hack/tools
 
-go 1.17
+go 1.18
 
 require (
-	github.com/golangci/golangci-lint v1.45.1
+	github.com/golangci/golangci-lint v1.45.2
 	github.com/goreleaser/goreleaser v1.6.1
```

Comment:
Do we need to also bump goreleaser to a version that supports Go 1.18? Looking at their [releases](https://github.com/goreleaser/goreleaser/releases), it seems like the v1.7.0+ tags support Go 1.18. The release workflow, which is responsible for running goreleaser under-the-hood appears to have passed, so we might be fine as long as the root go.mod specified as Go 1.18 version? Thoughts:?

Reply:
goreleaser 1.8.3 is on go 1.18 too. We can bump it to the latest (1.8.3) too

Reply:
I think we also need to re-run `go mod tidy` in `hack/tools`. I'd expect to see some `hack/tools/go.sum` changes show up.

Reply:
I've bumped goreleaser to 1.8.3 and re-ran `go mod tidy` in `hack/tools`

Reply:
I've bumped goreleaser to 1.8.3 and re-ran `go mod tidy` in `hack/tools`